### ISSUE TITLE
Feature Request: description/notes per entry?

### ISSUE THREAD

**Issue by adamwolf** [ArchiveBox/ArchiveBox#660]
## Type 

 - [X] General question or discussion
 - [ ] Propose a brand new feature
 - [ ] Request modification of existing behavior or design

## What is the problem that your feature request solves
I need to remember why I added something :)

## Describe the ideal specific solution you'd want, and whether it fits into any broader scope of changes
What are your thoughts on adding a description or notes field?  Is this something you are really opposed to? Is this something other people are asking for?  If I were to add it, would it be reasonable to add now vs after some changes in-flight?

## How badly do you want this new feature?

 - [ ] It's an urgent deal-breaker, I can't live without it
 - [X] It's important to add it in the near-mid term future
 - [X] It would be nice to have eventually

---

 - [X] I'm willing to contribute [dev time](https://github.com/ArchiveBox/ArchiveBox#archivebox-development) / [money](https://github.com/sponsors/pirate) to fix this issue
 - [X] I like ArchiveBox so far / would recommend it to a friend
 - [ ] I've had a lot of difficulty getting ArchiveBox set up

**Comment by pirate** [comment]
I would not be opposed to adding it but I don't know how it's going to fit, we don't have much new column space available in the list views.

In general you may prefer a different tool like Archivy if you want more notes-centric archiving.

**Comment by adamwolf** [comment]
Yeah, I get the spacing thing.  Hmm. I'd personally be OK if it was only in
the detail view.

I'm currently abusing tags for this (and probably could keep doing that
forever).

**Comment by pirate** [comment]
If you want to add a field and make it visible in the detail view only to start that's fine by me. We can figure out a way to squeeze in in the list views / elsewhere later.

**Comment by karlicoss** [comment]
Wanted something similar, but at the `archivebox add` time (I have different 'sources', so would like to tag with a source). Is there something like that? From a quick look at the code looks that tags aren't exposed to the CLI, only in the web interface and some specific link extractors?

**Comment by pirate** [comment]
Yeah an `archivebox add --tag=tag1,tag2,tag3` option is a great idea. I just added it to v0.6, as well as a form to the add page to add tags with URLs:

- [`fea0b89` (#655)](https://github.com/ArchiveBox/ArchiveBox/pull/655/commits/fea0b89dbe7a31163ada86c9c99b2902d9e5df9a)
- [`f3a3d76` (#655)](https://github.com/ArchiveBox/ArchiveBox/pull/655/commits/f3a3d76439ccfb822197d086c94520686756aa5a)

<img width="45%" src="https://user-images.githubusercontent.com/511499/112715186-5a4aac00-8eb5-11eb-9b9d-8052db466188.png" align="top"/><img width="45%" src="https://user-images.githubusercontent.com/511499/112715241-967e0c80-8eb5-11eb-94b5-b5dfb915238b.png" align="top"/>
